app/src/main/java/com/x8bit/bitwarden/data/platform/processor/AuthenticatorBridgeProcessorImpl.kt

```diff
@@ -110,14 +117,33 @@ class AuthenticatorBridgeProcessorImpl(
         }
 
         override fun startAddTotpLoginItemFlow(data: EncryptedAddTotpLoginItemData): Boolean {
-            val symmetricEncryptionKey = symmetricEncryptionKeyData ?: return false
+            val symmetricEncryptionKey = symmetricEncryptionKeyData ?: run {
+                Timber.e(
+                    t = IllegalStateException(),
+                    message = "Unable to start add TOTP item flow when " +
+                        "symmetricEncryptionKeyData is null.",
+                )
+                return false
+            }
             val intent = createAddTotpItemFromAuthenticatorIntent(context = applicationContext)
             val totpData = data.decrypt(symmetricEncryptionKey)
+                .onFailure {
+                    Timber.e(
+                        t = IllegalStateException(),
+                        message = "Unable to decrypt TOTP data.",
+                    )
+                }
                 .getOrNull()
                 ?.totpUri
                 ?.toUri()
                 ?.getTotpDataOrNull()
-                ?: return false
+                ?: run {
+                    Timber.e(
+                        t = IllegalStateException(),
+                        message = "Unable to parse TOTP URI.",
```

Comment:
So this error _could_ get logged twice, once here and possibly once if the decryption was the error. Is that the intention?

Reply:
Unlikely. Adding a `return` to the `onFailure` block.